GenumKit/Stencil/Filters.swift

```diff
@@ -106,3 +126,83 @@ struct NumFilters {
     return "\(percent)%"
   }
 }
+
+struct ModelParsingFilters {
```

Comment:
`equalityList`. Basically, the same problem as above. Need to create an array of `==` statements, and there is no `map` function.